api/ipfsproxy/ipfsproxy.go

```diff
@@ -475,6 +475,7 @@ func (proxy *Server) pinLsHandler(w http.ResponseWriter, r *http.Request) {
 					Type: pin.Mode.ToIPFSPinStatus(),
 				}
 				resBytes, _ := json.Marshal(ipinfo)
+				resBytes = append(resBytes, "\n"...)
 				w.Write(resBytes)
```

Comment:
```suggestion
				w.Write(resBytes)
				w.Write("\n")
```

Reply:
I think this is slightly better to avoid having to extend the resBytes slice if it's at capacity.

Reply:
yeah, sorry, editing in Github. Trying again.

Reply:
> Just thought it wouldn't matter here as it always only contains a single entry. But as said, I don't know much about go, yet.

I'm not 100% sure, but the slice might have the same size as the number of bytes of the json entry. Appending one byte to it might mean increasing capacity and causing a larger allocation. Not sure. It just feels more idiomatic this way.

Reply:
Sure. For me this looks good now. :-)